src/deep-resource/lib/Resource/Action.js

```diff
@@ -20,8 +20,9 @@ export class Action {
    * @param {Object} source
    * @param {String} region
    * @param {Boolean} forceUserIdentity
+   * @param {Object} apiCache
    */
-  constructor(resource, name, type, methods, source, region, forceUserIdentity) {
+  constructor(resource, name, type, methods, source, region, forceUserIdentity, apiCache) {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [This function has too many parameters. (8)](https://www.codacy.com/app/MitocGroup/deep-framework/file/1612096741/issues/source?bid=2895475&fileBranchId=3028940#l25)